src/components/Counter.js

```diff
@@ -21,23 +25,23 @@ class Counter extends Component {
             <p>
                 Clicked: {this.props.count} times
                 {" "}
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment() }>
```

Comment:
No need for callback here if immediately return function.  The function will be invoked automatically when the event is triggered. 
`onClick={this.props.increment}`